src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1503,6 +1503,11 @@ private void AddAdditionalTraceContext(MapField<string, string> attributes, Scri
                 {
                     attributes[ScriptConstants.LiveLogsSessionAIKey] = sessionid;
                 }
+                string operation_name = context.FunctionMetadata.Name;
```

Comment:
Why is the function name being populated into an OperationName property?

Reply:
The value of the operation name included as a property of all traces emitted by the host is the function name, as seen here: 
https://github.com/Azure/azure-webjobs-sdk/blob/16d138edf3eda4663f8ab8bb7aee4a7645fcbfe4/src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs#L185C114-L185C114